08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,137 @@
+body {
+  font-family:arial, sans-serif;
+}
+
+header {
+  text-align: center;
+}
+
+header p {
+  margin: 20px 256px;
+  text-align: center;
+}
+
+main {
+  width: 100%;
```

Comment:
if width is 100% then where is the 1.6% margin going to go? Remember, the whole of the content should be 100% or less